### PR TITLE
Integrate e-Reg API for Registartion

### PR DESCRIPTION
**Pull request by abs51295**
Resolves Issue #6 .

For UI changes please have a look at this video: https://goo.gl/1TkTBE

### CODE REVIEW

**Comment by abs51295** [comment]
@chhavip @riddhik84 I had issues with rebasing so I performed merge instead. You might want to squash commits while merging. Please review and suggest changes if any.

**Comment by chhavip** [comment]
@abs51295 Please include UI here as well for other participants to view

File: app/src/main/java/com/systers/conference/callback/ObjectResponseCallback.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.systers.conference.callback;
+
+
```
Comment (chhavip) [inline app/src/main/java/com/systers/conference/callback/ObjectResponseCallback.java]: remove extra line

File: app/src/main/java/com/systers/conference/model/Page.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package com.systers.conference.model;
+
```
Comment (chhavip) [inline app/src/main/java/com/systers/conference/model/Page.java]: remove extra lines

File: app/src/main/java/com/systers/conference/model/Page.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import com.google.gson.annotations.SerializedName;
+
+public class Page {
+    @SerializedName("pageid")
+    private String pageId;
+
+    @SerializedName("page")
+    private String pageName;
+
```
Comment (chhavip) [inline app/src/main/java/com/systers/conference/model/Page.java]: remove extra lines

**Comment by MeepyMay** [comment]
@abs51295 Has the changes been completed?

**Comment by abs51295** [comment]
@chhavip @MeepyMay changes have been completed.

**Pull request merged**